Gem/Code/Source/RHI/TrianglesConstantBufferExampleComponent.cpp

```diff
@@ -87,11 +87,8 @@ namespace AtomSampleViewer
             colorMultiplier.StoreToFloat4(&trianglesData[triangleIdx].m_colorMultiplier[0]);
         }
 
-        // Calculate the alignment
-        const uint32_t alignment = RHI::AlignUp(static_cast<uint32_t>(sizeof(InstanceInfo)), m_constantBufferAlighment);
-
         // All triangles data will be uploaded in one go to the constant buffer once per frame.
-        UploadDataToConstantBuffer(trianglesData, alignment, s_numberOfTrianglesTotal);
+        UploadDataToConstantBuffer(trianglesData, static_cast<uint32_t>(sizeof(InstanceInfo)), s_numberOfTrianglesTotal);
```

Comment:
I think this was added by Michael to address issues with instancing on AMD machines. Are you able to test that this isn't breaking this sample on the AMD machine?

Reply:
I tested on the AMD machine, both vulkan and dx12, with no issues. The size of the buffer is already s_numberOfTrianglesTotal * sizeof(InstanceInfo), and the offset is 0, so the alignment wasn't really doing anything here other than mapping more data than what's actually present in trianglesData.